samples/PSFScriptingSample/PSFScriptingSample/SignMSIX.cmd

```diff
@@ -0,0 +1,4 @@
+﻿@echo off
+setlocal
+set PATH=%PATH%;C:\Program Files (x86)\Windows Kits\10\bin\10.0.17763.0\x64
```

Comment:
>17763 [](start = 64, length = 5)

This WDK is very specific and doesn't align with the any other WDK requirements in this project. We should find a way to grab the WDK that exists on the system.